src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -352,7 +352,7 @@ class F extends CopyToTextureUtils {
 
     // The source canvas has bgra8unorm back resource. We
     // swizzle the channels to align with 2d/webgl canvas and
-    // clear alpha to opaque when context compositingAlphaMode
+    // clear alpha to opaque when context alphaMode
     // is set to opaque (follow webgpu spec).
```

Comment:
compositingAlphaMode wasn't supposed to affect the results of readbacks. I'm unclear about what this comment was talking about when it says "follow webgpu spec".

It seems like these functions are passing through a `premultiplied: boolean` when what they really mean is `alphaMode: GPUCanvasAlphaMode`, but I'm not sure about that. I understood this when I reviewed it but I don't anymore.

Can you investigate and maybe clarify the comments and function parameters where it makes sense?

Reply:
Change `premultiplied: boolean` to `alphaMode: GPUCanvasAlphaMode` for gpu context canvas. (Preserve those for 2d, webgl, and related utils)